backend/src/main/java/business/AlunoExcel.java

```diff
@@ -42,18 +97,441 @@ public HSSFWorkbook gerarExcel() throws SQLException {
+				cf.createCell(56).setCellValue(String.valueOf(mf.get(i).getDataNascimento()));
+				cf.createCell(57).setCellValue(mf.get(i).getEscolaridade());
+				cf.createCell(58).setCellValue(mf.get(i).getOcupacao());
+				cf.createCell(59).setCellValue(mf.get(i).getCondicaoTrabalho());
+				cf.createCell(60).setCellValue(String.valueOf(mf.get(i).getSalario()));
+				cf.createCell(61).setCellValue(mf.get(i).getLocalTrabalho());
+			} else {
+				cf.getCell(54).setCellValue(cf.getCell(54).getStringCellValue().concat(quebraLinha + mf.get(i).getNome()));
+				cf.getCell(55).setCellValue(cf.getCell(55).getStringCellValue().concat(quebraLinha + mf.get(i).getParentesco()));
+				cf.getCell(56).setCellValue(cf.getCell(56).getStringCellValue().concat(quebraLinha + String.valueOf(mf.get(i).getDataNascimento())));
+				cf.getCell(57).setCellValue(cf.getCell(57).getStringCellValue().concat(quebraLinha + mf.get(i).getEscolaridade()));
+				cf.getCell(58).setCellValue(cf.getCell(58).getStringCellValue().concat(quebraLinha + mf.get(i).getOcupacao()));
+				cf.getCell(59).setCellValue(cf.getCell(59).getStringCellValue().concat(quebraLinha + mf.get(i).getCondicaoTrabalho()));
+				cf.getCell(60).setCellValue(cf.getCell(60).getStringCellValue().concat(quebraLinha + String.valueOf(mf.get(i).getSalario())));
+				cf.getCell(61).setCellValue(cf.getCell(61).getStringCellValue().concat(quebraLinha + mf.get(i).getLocalTrabalho()));
+			}
+		}
+	}
+	
+	public void preencherSituacaoHabitacional(HSSFSheet sheet, int linha, AlunoDTO aluno) {
+		HSSFRow rowHead = sheet.getRow(1);
+		rowHead.createCell(62).setCellValue("Situação do Imóvel");
+		rowHead.createCell(63).setCellValue("Nº de Comodos");
+		rowHead.createCell(64).setCellValue("Esgoto?");
+		rowHead.createCell(65).setCellValue("Rede Elétrica?");
+		rowHead.createCell(66).setCellValue("Asfalto?");
+		rowHead.createCell(67).setCellValue("Alvenaria?");
+		rowHead.createCell(68).setCellValue("Casa de Madeira?");
+		rowHead.createCell(69).setCellValue("Imovel Construido em Área Irregular?");
+		
+		HSSFRow shLinha = sheet.getRow(linha);
+		
+		SituacaoHabitacional sh = aluno.getSituacaoHabitacional();
+		
+		shLinha.createCell(62).setCellValue(sh.getSituacao());
+		shLinha.createCell(63).setCellValue(String.valueOf(sh.getNumeroComodos()));
+		shLinha.createCell(64).setCellValue(String.valueOf(sh.getEsgoto()));
+		shLinha.createCell(65).setCellValue(String.valueOf(sh.getRedeEletrica()));
+		shLinha.createCell(66).setCellValue(String.valueOf(sh.getAsfalto()));
+		shLinha.createCell(67).setCellValue(String.valueOf(sh.getAlvenaria()));
+		shLinha.createCell(68).setCellValue(String.valueOf(sh.getMadeira()));
+		shLinha.createCell(69).setCellValue(String.valueOf(sh.getAreaIrregular()));
+	}
+	
+	public void preencherAparelhosEletronicos(HSSFSheet sheet, int linha, AlunoDTO aluno) {
+		HSSFRow rowHead = sheet.getRow(1);
+		rowHead.createCell(70).setCellValue("Televisão");
```

Comment:
Onde for dados boolean, fazer um if para imprimir Sim ou Não, ao inves de true ou false